packages/calling/src/Metrics/index.ts

```diff
@@ -285,7 +285,10 @@ class MetricManager implements IMetricManager {
           },
           fields: {
             device_url: this.deviceInfo?.device?.clientDeviceUri,
-            calling_sdk_version: process.env.CALLING_SDK_VERSION || VERSION,
+            calling_sdk_version:
+              process && process.env.CALLING_SDK_VERSION
+                ? process.env.CALLING_SDK_VERSION || VERSION
```

Comment:
Nit: Just a small change - 
```suggestion
                ? process.env.CALLING_SDK_VERSION
```
If we have the sdk version existing and is not undefined, we ca use it. Else we can fallback on VERSION.

Reply:
Taken care of it.  Thank you.